subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -3237,6 +3237,7 @@ SHELL_SUBCMD_ADD((wifi), connect, &wifi_commands,
 		  "[-I, --eap-id1]: Client Identity. Default no eap identity.\n"
 		  "[-P, --eap-pwd1]: Client Password.\n"
 		  "Default no password for eap user.\n"
+		  "[-R, --11r]: Use IEEE80211R fast BSS transition connect."
```

Comment:
At least in the long options list the long option name seems to be `--ieee-80211r` and not `--11r`
https://github.com/zephyrproject-rtos/zephyr/blob/afce5329f772a4306af699c7a06efddf93cadca0/subsys/net/l2/wifi/wifi_shell.c#L584

Reply:
fixed